include/solvers/navier_stokes_scratch_data.h

```diff
@@ -603,24 +604,30 @@ class NavierStokesScratchData : public PhysicsScratchDataBase
    * @brief enable_particle_field_projection Enables the collection of the particle fields
    * projection data by the scratch
    *
-   * @param quadrature Quadrature rule of the Navier-Stokes problem assembly
+   * @param[in] quadrature Quadrature rule of the Navier-Stokes problem assembly
    *
-   * @param mapping Mapping used for the Navier-Stokes problem assembly
+   * @param[in] mapping Mapping used for the Navier-Stokes problem assembly
+   *
+   * @param[in] fe_particle_drag_proj FiniteElement associated with the
+   * projected particle drag force
    *
-   * @param fe_particle_drag_proj FiniteElement associated with the projected particle drag force
+   * @param[in] fe_particle_two_way_coupling_force_proj FiniteElement associated
+   * with the projected particle two-way coupling force
    *
-   * @param fe_particle_two_way_coupling_force_proj FiniteElement associated with the projected
-   * particle two-way coupling force
+   * @param[in] fe_particle_velocity_proj FiniteElement associated with the
+   * projected particle velocity
    *
-   * @param fe_particle_velocity_proj FiniteElement associated with the projected particle velocity
+   * @param[in] fe_particle_velocity_proj FiniteElement associated with the
```

Comment:
You have comments / documentation for twice the same parameters. It should be velocity_proj then momentum transfer coefficient.